db/migrations/20190723233130_create_polymorphic_belongs_to.cr

```diff
@@ -0,0 +1,28 @@
+class CreatePolymorphicBelongsTo::V20190723233130 < Avram::Migrator::Migration::V1
+  def migrate
+    create table_for(PolymorphicTask) do
+      primary_key id : Int64
+      add_timestamps
+      add title : String
+    end
+
+    create table_for(PolymorphicTaskList) do
+      primary_key id : Int64
+      add_timestamps
+      add title : String
+    end
+
+    create table_for(PolymorphicEvent) do
+      primary_key id : Int64
+      add_timestamps
+      add_belongs_to task : PolymorphicTask?, on_delete: :cascade
+      add_belongs_to task_list : PolymorphicTaskList?, on_delete: :cascade
```

Comment:
Should there be a `has_many events` for the Task and List?

Reply:
There doesn’t have to be if you don’t use it. In real life you probably would have that but isn’t necessary for the test since we test has many elsewhere